nautobot_ssot/templates/nautobot_ssot/sync_detail.html

```diff
@@ -120,6 +129,77 @@
                     </tr>
                 </table>
             </div>
+            {% if object.load_source_memory_size %}
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>Memory Usage Stats</strong>
+                </div>
+                <table class="table table-hover panel-body attr-table">
+                    <tr>
+                        <td>Memory used loading {{ object.source }}</td>
+                        <td>
+                            <table class="table table-hover panel-body attr-table">
+                                <tr>
+                                    <td>Final</td>
+                                    <td>{{ object.load_source_memory_size | intcomma }} bytes</td>
+                                </tr>
+                                <tr>
+                                    <td>Peak</td>
+                                    <td>{{ object.load_source_memory_size | intcomma }} bytes</td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Memory used loading {{ object.target }}</td>
+                        <td>
+                            <table class="table table-hover panel-body attr-table">
+                                <tr>
+                                    <td>Final</td>
+                                    <td>{{ object.load_target_memory_size | intcomma }} bytes</td>
```

Comment:
Using `intcomma` is an improvement but the values are still hard to read as raw byte numbers, I think. Maybe we should actually add a custom template filter (similar to Nautobot core's `humanize_speed`) that would render values as "11.42 MiB", etc. rather than "11,420,190 bytes"?

Reply:
understood